src/js/cljs.js

```diff
@@ -460,6 +460,17 @@ async function startClojureScriptEngine(opts: CLIOptsType): Promise<mixed> {
 
     startREPL(opts);
   }
+
+  if (!mainNsName && !repl) {
+    initClojureScriptEngine(opts);
+    execute(
+      "(require 'lumo.repl) (lumo.repl/run-main-cli-fn)",
```

Comment:
It does need to require, I tried without and It was failing. About the JS function it could be done also in another PR as refinement, but if just a wrapper, I am not sure about the value..